lib/init/features/project.js

```diff
@@ -53,9 +53,11 @@ module.exports = function(setup, config) {
         return;
       }
       projectId = prompt.listLabelToValue(projectId, choices);
+      var instance = projects[projectId].instances.database[0];
 
       // write "default" alias and activate it immediately
       _.set(setup.rcfile, 'projects.default', projectId);
+      _.set(setup.rcfile, 'projects.default_instance', instance);
```

Comment:
`setup` is just an object that hangs around during init to remember things. In this case, we don't want to actually persist the instance into the rcfile (that's actually creating a second alias named `default_instance`, we just want to keep track of it for later. Just doing `setup.instance = instance` and then later retrieving it via `setup.instance` should get you the same functionality without the undesired side effects..